app/code/core/Mage/Newsletter/Block/Widget/Subscribe.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * OpenMage
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available at https://opensource.org/license/osl-3-0-php
+ *
+ * @category   Mage
+ * @package    Mage_Newsletter
+ * @copyright  Copyright (c) 2006-2020 Magento, Inc. (https://www.magento.com)
+ * @copyright  Copyright (c) 2024 The OpenMage Contributors (https://www.openmage.org)
+ * @license    https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+/**
+ * Newsletter subscribe form Widget
+ *
+ * @category   Mage
+ * @package    Mage_Newsletter
+ * @method string getSuccessMessage()
+ * @method string getErrorMessage()
+ * @method string getFormActionUrl()
+ */
+class Mage_Newsletter_Block_Widget_Subscribe extends Mage_Newsletter_Block_Subscribe implements Mage_Widget_Block_Interface
+{
+    protected function _construct()
+    {
+        parent::_construct();
+    }
```

Comment:
```suggestion
```

is the constructor needed since it just calls the parent?

Reply:
I tested the PR without this constructor and it works fine

Reply:
I had kept it for safety to ensure the same behavior as the block if something is added in the future, but it's not that important

Reply:
If I remember correctly we had a PR some months ago removing these kinds of constructors, but I see your point, let's think about that for a minute and see

Reply:
I would vote to remove an 'empty' constructor method like this, especially if that is consistent with the rest of the code base. If we need to modify anything here in the future it should be simple enough to add back in.

Reply:
drop it